src/c_buttons.hpp

```diff
@@ -280,6 +280,15 @@ void wifiLoop() {
 }
 #endif
 
+#if !defined(OAT_DEBUG_BUILD) && defined(SUPPORT_SERIAL_CONTROL)
+void serialLoop() {
+    meadeSerialStreamReader.loop();
+}
+#else
+void serialLoop() {
+
```

Comment:
No serial in a DEBUG build?

Reply:
This was existing functionality that I just moved over. I think the debugger takes over uart0 anyway